angular-swfobject.js

```diff
@@ -60,23 +60,21 @@ angular.module('swfobject', [])
                   $interval.cancel(loadCheckInterval);
                   loadCheckInterval = null;
                   //Execute function
-                  fn();
+                  fn({evt: evt});
```

Comment:
unfortunately this style not work and will be passed as undefined to callback function [as](http://stackoverflow.com/questions/17556703/angularjs-directive-call-function-specified-in-attribute-and-pass-an-argument-to)